.travis.yml

```diff
@@ -59,15 +63,25 @@ install:
     fi
   - pip install -r requirements_dev.txt
   - python setup.py install
-  - python setup.py build_ext --inplace
-  - if [ ${TEST_DIR} = "tests/docs" ]; then
-      pip install sphinx_rtd_theme ;
+  - if [ "$TEST_DIR" = "$DEPLOY_DIR" -a ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then
+      if [ ! -d ${HOME}/google-cloud-sdk ]; then curl https://sdk.cloud.google.com | bash; fi ;
+      openssl aes-256-cbc -K $encrypted_7e0a8632ac3f_key -iv $encrypted_7e0a8632ac3f_iv -in credentials.tar.gz.enc -out credentials.tar.gz -d ;
+      tar -xzf credentials.tar.gz ;
+      gcloud auth activate-service-account --key-file credentials/client-secret-docs.json ;
+      gsutil cp gs://simpeg/_build.tar.gz $PWD/docs/ ;
+      mkdir $PWD/docs/_build ;
+      tar -xzf $PWD/docs/_build.tar.gz -C $PWD/docs/_build ;
+      ls $PWD/docs/_build/ ;
     fi
 
 # Run test
 script:
-  - nosetests $TEST_DIR --with-cov --cov SimPEG --cov-config .coveragerc -v -s;
-
+  # - nosetests $TEST_DIR --with-cov --cov SimPEG --cov-config .coveragerc -v -s
+  - if [ "$TEST_DIR" = "$DEPLOY_DIR" -a ${TRAVIS_PYTHON_VERSION:0:1} == "3" ]; then
```

Comment:
on python 3, we only check that the docs build - the examples are not run (they take too long at the moment)